htslib.mk

```diff
@@ -188,7 +188,8 @@ $(HTSDIR)/htslib.pc.tmp:
 #
 #	clean: clean-htslib
 
-all-htslib clean-htslib install-htslib plugins-htslib:
+all-htslib check-htslib clean-htslib distclean-htslib install-htslib plugins-htslib test-htslib:
```

Comment:
Should we also have check-all test-all and distclean-all targets, in a similar vein to clean-all?
"distclean-all" is the one I miss the most when testing releases.

Reply:
Those would need to be added to the samtools and bcftools Makefiles.  The parts here are the bits that they reference and need to be in place first.

Reply:
Ah yes my mistake.